spec/setup_acceptance_node.pp

```diff
@@ -1,5 +1,5 @@
-if $facts['os']['name'] == 'CentOS' {
-  package { 'epel-release':
+if $facts['os']['family'] == 'RedHat' and $facts['os']['name'] != 'Fedora' {
+    package { 'epel-release':
```

Comment:
Please be careful with indenting. I'm surprised CI didn't flag this
```suggestion
  package { 'epel-release':
```

Reply:
@ekohl Fixed it here! https://github.com/theforeman/puppet-dhcp/pull/238